pkg/isbsvc/jetstream_service.go

```diff
@@ -262,6 +293,11 @@ func (jss *jetStreamSvc) ValidateBuffersAndBuckets(ctx context.Context, buffers,
 			return fmt.Errorf("failed to query side inputs store KV %q, %w", sideInputsKVName, err)
 		}
 	}
+	if servingSourceStream != "" {
+		if _, err := js.StreamInfo(servingSourceStream); err != nil {
```

Comment:
Just confirm - we are not doing validation in any place at this moment, right? Could be added to the vertex init container when it's a serving source.

Reply:
let's list down all the validations we have to do.